src/styles/box/stars.scss

```diff
@@ -0,0 +1,49 @@
+.product-card {
+  &__stars {
+    box-sizing: border-box;
+    margin: 0 4px 0 3px;
+    padding: 0;
+  }
+
+  &__star {
+    display: inline-block;
+    background-image: $star-bg;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: $star-width;
```

Comment:
The variable `$star-width` is used but not defined in this file. Ensure that it is defined elsewhere in your SCSS files, or define it here to avoid compilation errors.